### PR TITLE
Update inforamation about provided wheels versions

### PR DESCRIPTION
**Pull request by Czaki**
This PR is a follow-up for #559 and adds information about available wheels in the documentation.  

It may be a little too detailed?

### CODE REVIEW

File: doc/misc.rst [hunk 1]
```diff
@@ -133,17 +133,24 @@ and it is packaged so that the OpenCL drivers that are registered in the OS
...
 
 
-Installing from PyPI with Linux wheels
---------------------------------------
+Installing from PyPI wheels
+---------------------------
+
+PyOpenCL distributes wheels for Python 3.6-3.10 for macOS, Windows, GLIBC base Linux and musl base Linux.
+For GLIBC base linux minimum version of GLIBC is 2.17. 
+There are also distributted wheels for PyPy 3.7-3.9 but only for GLIBC based linux
```
Comment (inducer) [inline doc/misc.rst]: I agree that this information will likely not age well, especially the reference to specific platforms and version numbers. Maybe just link to the pypi page? People can see themselves whether a wheel is available for their platform. I think less specific is the way to go here.

File: doc/misc.rst [hunk 2]
```diff
@@ -133,17 +133,24 @@ and it is packaged so that the OpenCL drivers that are registered in the OS
...
-GLIBC>=2.5 based distributions.
 
-On Linux, type::
+To install type::
 
     pip install pyopencl
 
-The wheels comes with OCL-ICD bundled and configured to use any OpenCL implementation
+On Linux the wheels comes with OCL-ICD bundled and configured to use any OpenCL implementation
 supporting ICD interface installed in :file:`/etc/OpenCL/vendors`
```
Comment (inducer) [inline doc/misc.rst]: Integrate this with the blurb about loaders above?

**Review by inducer — COMMENTED** [review]
Thanks! See below for some comments.

**Comment by Czaki** [comment]
> Thanks! See below for some comments.

I try to address your remarks. English is not my first language so I will be happy to any remark.

There is an indentation error that breaks zenodo badge. I may add a fix for thin in this PR if you want.

**Comment by inducer** [comment]
> There is an indentation error that breaks zenodo badge. I may add a fix for thin in this PR if you want.

Thanks for the tip. Already done: 27e9d5dd0c4ac421efe8a30a60f4e16892df0ab9

File: doc/misc.rst [hunk 3]
```diff
@@ -133,18 +133,22 @@ and it is packaged so that the OpenCL drivers that are registered in the OS
 using registry keys are automatically available.
 
 
-Installing from PyPI with Linux wheels
---------------------------------------
+Installing from PyPI wheels
+---------------------------
+
+PyOpenCL distributes wheels for most popular systems and versions.
+To check available versions please visit `PyPI page for PyOpenCL <https://pypi.org/project/pyopencl/>`_
```
Comment (inducer) [inline doc/misc.rst]:
```suggestion
PyOpenCL distributes wheels for most popular OSs and Python versions.
To check available versions please visit `PyPI page for PyOpenCL <https://pypi.org/project/pyopencl/>`__.
```

File: doc/misc.rst [hunk 4]
```diff
@@ -133,18 +133,22 @@ and it is packaged so that the OpenCL drivers that are registered in the OS
...
+
 
-PyOpenCL distributes ``manylinux1`` wheels in PyPI. These wheels are compatible with
-GLIBC>=2.5 based distributions.
+On Linux the wheels comes with OCL-ICD bundled and configured to use any OpenCL implementation
+supporting ICD interface installed in :file:`/etc/OpenCL/vendors`.
+Wheels for Windows and MacOS are build using 
+OpenCL-ICD-Loader from KhronosGroup
```
Comment (inducer) [inline doc/misc.rst]:
```suggestion
On Linux, the wheels come with `OCL-ICD <https://github.com/OCL-dev/ocl-icd>`__ bundled and
configured to use any OpenCL implementation supporting the ICD interface and listed
in :file:`/etc/OpenCL/vendors`.
Wheels for Windows and MacOS are built using the ICD Loader from the Khronos Group.
```

File: doc/misc.rst [hunk 5]
```diff
@@ -133,18 +133,22 @@ and it is packaged so that the OpenCL drivers that are registered in the OS
...
 
-On Linux, type::
+To install type::
```
Comment (inducer) [inline doc/misc.rst]:
```suggestion
To install, type::
```

**Comment by inducer** [comment]
Thx!

**Pull request merged**